pkg/admission/kubequota/kubequota_admission.go

```diff
@@ -129,8 +132,18 @@ var _ = initializers.WantsServerShutdownChannel(&KubeResourceQuota{})
 // Validate gets or creates a resourcequota.QuotaAdmission plugin for the logical cluster in the request and then
 // delegates validation to it.
 func (k *KubeResourceQuota) Validate(ctx context.Context, a admission.Attributes, o admission.ObjectInterfaces) error {
+	if a.GetResource() == corev1alpha1.SchemeGroupVersion.WithResource("logicalclusters") {
```

Comment:
These checks are:
* don't check quota on logicalclusters
* don't check quota on crbs named workspace-admin, workspace-admin-legacy and workspace-access-legacy

? Should we comment why?

Reply:
done